tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/species_features.tsx

```diff
@@ -20,61 +20,63 @@ export const feature_mcolor3: Feature<string> = {
 
 export const flavor_text: Feature<string> = {
   name: 'Flavor Text',
-  description: 'Describe your character!',
+  description:
+    "Appears when your character is examined (but only if they're identifiable - try a gas mask).",
   component: FeatureTextInput,
 };
 
 export const silicon_flavor_text: Feature<string> = {
   name: 'Flavor Text (Silicon)',
-  description: 'Describe your cyborg/AI shell!',
+  description: "Only appears if you're playing as a borg/AI.",
   component: FeatureTextInput,
 };
 
 export const ooc_notes: Feature<string> = {
   name: 'OOC Notes',
-  description: 'Talk about your character OOCly!',
+  description:
+    'Primarily for things like ERP, general, antag/whatever preferences.',
```

Comment:
```suggestion
    'You can put any relevant out-of-character notes here.',
```
I think it goes without saying that it can include that, we don't need to be explicit here.

Reply:
the suggested replacement is just irrelevant info since its already implied, so ill just remove the tooltip